src/Build/BuildCheck/API/IBuildCheckRegistrationContext.cs

```diff
@@ -9,4 +9,5 @@ public interface IBuildCheckRegistrationContext
 {
     void RegisterEvaluatedPropertiesAction(Action<BuildCheckDataContext<EvaluatedPropertiesAnalysisData>> evaluatedPropertiesAction);
     void RegisterParsedItemsAction(Action<BuildCheckDataContext<ParsedItemsAnalysisData>> parsedItemsAction);
+    void RegisterTaskInvocationAction(Action<BuildCheckDataContext<TaskInvocationAnalysisData>> taskInvocationAction);
```

Comment:
I am looking for feedback on the name. This could for example be "execution" instead of "invocation": `RegisterTaskExecutionAction`.

Reply:
This feels nicely descriptive - I'd leave as is